docs: add ground segmentation (#63)
tier4/driving_log_replayer_v2: docs/use_case/ground_segmentation.en.md

# Evaluate Ground Segmentation

> Evaluate the function of segmenting and removing ground point clouds from input point clouds.

Reviewer: ```suggestion
Evaluate the performance of the Obstacle Segmentation sub-component in Autoware, which is responsible for identifying point clouds originating from obstacle that vehicle should avoid.
```

## Ground Truth data

The Ground Truth data required for evaluation can be provided using the following two methods, and each can be used by changing the `Evaluation.Conditions.Method` of the scenario.

### annotated_rosbag

This method involves adding a field to the point cloud data in the bag file to represent semantic labels.

> Synchronize and subscribe to topics before and after ground removal, and evaluate accuracy by comparing the number of points with ground labels.

Reviewer: ```suggestion
Synchronize and subscribe to topics before and after ground removal, and evaluate the accuracy by comparing the number of points with ground and non-ground labels.
```

In this evaluation framework, the semantic labels are assumed to be recorded in an `INT32` field named `entity_id`.

### annotated_pcd

> This method involves adding a field to the point cloud data provided as a dataset (~/driving_log_replayer_v2/ground_segmentation/dataset/data/LIDAR_CONCAT/\*.pnd.bin) to represent semantic labels.

Reviewer: ```suggestion
This method involves adding a field to the point cloud data provided as a dataset (~/driving_log_replayer_v2/ground_segmentation/dataset/data/LIDAR_CONCAT/\*.pcd.bin) to represent semantic labels.
```

## Evaluation result

The results are calculated for each subscription. The format and available states are described below.

### Normal

If the Accuracy obtained through the evaluation meets or exceeds the Evaluation.Conditions.accuracy_min specified in the scenario, it is assumed to be normal.

### Error

When the normal condition is not met

## Topic name and data type used by evaluation node

Subscribed topics:

| topic name                                                | Data type                   |
| --------------------------------------------------------- | --------------------------- |
| /sensing/lidar/concatenated/pointcloud 　　               | sensor_msgs/msg/PointCloud2 |
| /perception/obstacle_segmentation/single_frame/pointcloud | sensor_msgs/msg/PointCloud2 |

**NOTE: the`/perception/obstacle_segmentation/single_frame/pointcloud`topic can be modified by changing the `evaluation_target_topic` launch argument.**

Published topics:

| topic name | Data type |
| ---------- | --------- |
| -          | -         |

## Arguments passed to logging_simulator.launch

- localization: false
- planning: false

> - control: false

Reviewer: ```suggestion
- control: false
- sensing: false
```
